src/Persister/EntityPersister.php

```diff
@@ -99,6 +106,23 @@ public function getUpdateQuery($object)
         return Statement::create($query, ['id' => $id, 'props' => $propertyValues]);
     }
 
+    /**
+     * Refreshes a managed entity.
+     *
+     * @param int  $id
+     * @param object $entity The entity to refresh.
+     *
+     * @return void
+     */
+    public function refresh($id, $entity)
```

Comment:
We do not need that because we call it like: 

```php
$this->getPersister(get_class($entity))->refresh($this->entityIds[spl_object_hash($entity)], $entity);
```

Actually. Maybe we should see if we can remove the second parameter here. Since it is not used.